slick.core.js

```diff
@@ -564,16 +580,18 @@
     }
 
     function cloneTreeColumns() {
-      return $.extend(true, [], treeColumns);
+      return extend([], treeColumns);
     }
 
     init();
 
     this.hasDepth = function () {
 
       for (var i in treeColumns)
+      {
```

Comment:
could you use the JS standard and move the opening curly brace on the same line of the `for() {`